### PR TITLE
Update MySQL dependencies

### PR DESCRIPTION
**Pull request by variadico**
Currently, these apps are failing with the new MySQL and Postgres packages. This updates the dependencies to newer versions to fix the failures.

@wallyqs @zquestz @johnSchnake

### CODE REVIEW

File: nats-ping/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+RUN apt-get update;apt-get -y install golang git
+ENV GOPATH=/go
+ENV PATH=$PATH:$GOPATH/bin
+RUN mkdir -p /go/bin
+WORKDIR /go
+ADD nats-ping.go /go/
+RUN go get |  true
+RUN go build nats-ping.go
+RUN chmod +x /go/nats-ping
+CMD ./nats-ping
```
Comment (wallyqs) [inline nats-ping/Dockerfile]: :+1:

**Comment by variadico** [comment]
Currently running system tests…

File: example-rails-mysql/Gemfile [hunk 2]
```diff
@@ -2,7 +2,7 @@ source 'https://rubygems.org'
 
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem 'rails', '4.1.6'
+gem 'rails'
```
Comment (wallyqs) [inline example-rails-mysql/Gemfile]: We should probably still lock here to `4.2.6` just in case

Comment (variadico) [inline example-rails-mysql/Gemfile]: @johnSchnake, does this grab the latest `rails` gem?

Comment (johnSchnake) [inline example-rails-mysql/Gemfile]: Yea putting rails will just take the latest; but the safest way is to put '4.2.6'.  Normally you would put '~> 4.2.6' but there the dependency chains make me :weary: so maybe just locking it is best for a sample app.

**Comment by variadico** [comment]
Tests passed before locking Rails. Running everything again after locking.

**Comment by wallyqs** [comment]
LGTM after green systests run

**Comment by lilirui** [comment]
Please trigger a test on AWS also!

**Comment by variadico** [comment]
Triggered.

File: example-rails-mysql/Gemfile.lock [hunk 3]
```diff
@@ -141,7 +168,7 @@ DEPENDENCIES
   jbuilder (~> 2.0)
   jquery-rails
   mysql2
-  rails (= 4.1.6)
+  rails
```
Comment (variadico) [inline example-rails-mysql/Gemfile.lock]: @wallyqs I think the problem is here.

**Comment by johnSchnake** [comment]
LGTM if green too (if you didnt already assume that)

**Pull request merged**